src/Containers/SparseAxisArray.jl

```diff
@@ -448,7 +464,6 @@ function Base.show(io::IOContext, x::SparseAxisArray)
         (i, (key, value)) in enumerate(x.data) if
         i < half_screen_rows || i > length(x) - half_screen_rows
     ]
-    sort!(key_strings; by = x -> x[1])
```

Comment:
I didn't run the full tests locally, but from what I can tell, there are few, if any, printing changes, because we used to sort the keys, and most tests/examples provided the keys in a nice sorted order.